src/vsr/replica.zig

```diff
@@ -10853,67 +10853,64 @@ pub fn ReplicaType(
             assert(self.grid_repair_budget_timeout.ticking);
             assert(self.grid.callback != .cancel);
             maybe(self.state_machine_opened);
-            if (!self.solo()) {
-                assert(self.repair_messages_budget_grid.available >=
-                    constants.grid_repair_request_max);
-            }
```

Comment:
Sorry,  I was unclear, I think we need both _this_ assert, and `assert(!self.solo());` below. The two asserts together are needed to justify that calling `decrement` is valid 


The `if (!self.solo)` one proves that we _can_ call decrement if we are not solo. The `assert(!self.solo)` proves that if we did call decrement, we are not solo.